TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Drivetrain.java

```diff
@@ -0,0 +1,36 @@
+package org.firstinspires.ftc.teamcode;
+
+import static com.qualcomm.robotcore.hardware.DcMotorSimple.Direction.FORWARD;
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.gamepad1;
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.hardwareMap;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+
+
+public class Drivetrain {
+    DcMotor moaterFL;
```

Comment:
Fix the spelling of all of the motors. Also this might be a little nit picky however I think its easier that we say what side the motor is on, then the location of the motor first. So in the case of front left it would be FrontLeftMotor rather than MotorFL